src/main/java/sirius/biz/codelists/LookupTable.java

```diff
@@ -124,7 +125,12 @@ public String normalizeCodeValue(@Nullable String code) {
             return Optional.empty();
         }
 
-        return performResolveName(normalizeCodeValue(code), lang);
+        try {
+            return performResolveName(normalizeCodeValue(code), lang);
+        } catch (Exception e) {
+            Exceptions.handle(e);
```

Comment:
Maybe we could fix this for IDBLookupValue as we could provide a finer grained error message and select the appropriate logger (Jupiter.LOG) - "Failed to lookup X in Y" ot the like.